product_docs/docs/biganimal/release/overview/05_database_version_policy.mdx

```diff
@@ -10,7 +10,7 @@ We support the major Postgres versions from the date they're made available unti
 | ---------------------------- | --------------------------------------------------- |
 | PostgreSQL                   | 11–15                                               |
 | EDB Postgres Advanced Server | 11–15, 14-15 for extreme-high-availability clusters |
-| EDB Postgres Extended Server | 12-15                                               |
+| EDB Postgres Extended Server | 12-15, 14-15 for extreme-high-availability clusters |
```

Comment:
I don't believe we actually support PGE for Single and HA offerings?  We will soon though?
```suggestion
| EDB Postgres Extended Server | 14-15 for extreme-high-availability clusters |
```

Reply:
Okay going to commit this